ci/steps/attach_logs.py

```diff
@@ -253,9 +252,24 @@ def upload_logs(
         only_failed=False,
     )
     if ok:
+        tasks = build_tasks.split(',')
+        build_dict = dict()
```

Comment:
stylistic nit - how about using a dict-comprehension:
```
build_dict = {
  task_name: logs.get_task_result(
    task_runs_dict=pipeline_run,
    task_ref_name=task_name,
    result_name='build_result'
  ) for task_name in tasks
}
```

also, maybe rather name it `task_logs` or sth. similar (reflecting the contents)?

Reply:
Agreed.